src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java

```diff
@@ -97,8 +100,42 @@ public static void rect(int left, int top, int width, int height, int color) {
 	}
 
 	/**
-	 * Draws a beveled, round rectangle that is substantially similar to default Minecraft UI panels.
+	 * Draws a rectangle for a Fluid, because fluids are tough.
 	 */
+	public static void rect(Fluid fluid, int left, int top, int width, int height, float uScale1, float vScale1, float uScale2, float vScale2, int color) {
```

Comment:
From the looks of it, uscale1/vscale1/uscale2/vscale2 are just u1/v1/u2/v2. Let's not complicate arguments.